Utils/azuretools-core/src/com/microsoft/azuretools/Constants.java

```diff
@@ -32,4 +32,5 @@
     public static String clientId = "61d65f5a-6e3b-468b-af73-a033f5098c5c";
```

Comment:
I don't think here is the best place for the string, global constants related with sign-in are located here. For this webapp related constant, let's put it into `/com/microsoft/intellij/runner/webapp/webappconfig` or `/com/microsoft/intellij/runner/webapp`

BTW, Remove the comments above this class for legal concern.